### PR TITLE
Node 10 support

### PR DESCRIPTION
**Pull request by hcourt**
Fixes https://github.com/mapbox/ecs-watchbot/issues/324

### CODE REVIEW

File: .artifacts.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ bundle:
     - filter: on-every-commit
       config:
         platform: nodejs
-        platform_version: '8.10'
+        platform_version: '10'
```
Comment (vsmart) [inline .artifacts.yml]: You could also opt to move to the default bundle, which is Node 10. That artifacts.yml would be:
```
version: v1
defaults: 
- bundle
```

File: package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "engines": {
-    "node": ">=8"
```
Comment (vsmart) [inline package.json]: Could you also update some of the Node dependencies as part of the release? E.g. mapbox/cloudfriend is 3 major versions behind. `npm outdated` will give a rundown of the major outdated ones.

Comment (hcourt) [reply package.json]: Doing so here: https://github.com/mapbox/ecs-watchbot/pull/326

**Comment by hcourt** [comment]
Tested with a staging stack.  Logs indicated that messages could be processed successfully.

**Comment by hcourt** [comment]
One (custom) resource was failing to finish creation, and I believe this is due to https://github.com/aws/aws-sdk-js/issues/2955

Trying out a fix for this.

**Comment by hcourt** [comment]
> One (custom) resource was failing to finish creation, and I believe this is due to [aws/aws-sdk-js#2955](https://github.com/aws/aws-sdk-js/issues/2955)
> 
> Trying out a fix for this.

This worked.

**Comment by hcourt** [comment]
Think this is finished.  Before releasing, I will also merge #326.

File: lib/template.js [hunk 3]
```diff
@@ -823,14 +823,14 @@ module.exports = (options = {}) => {
       Role: cf.getAtt(prefixed('LambdaScalingRole'), 'Arn'),
       Code: {
         ZipFile: cf.sub(`
-          const response = require('cfn-response');
+          const response = require('./cfn-response');
```
Comment (vsmart) [inline lib/template.js]: Wow - nice find, though that is super weird. I did see that in the support ticket you linked (https://github.com/aws/aws-sdk-js/issues/2955) they also recommend trying to update the AWS-SDK node dependency to the latest version. Would you consider trying that to see if it makes this change not needed?
We could also ship this as-is and open a support case to track - I'll leave that up to you.

Comment (hcourt) [reply lib/template.js]: Hm, going to try it out, shouldn't take that long.

Comment (hcourt) [reply lib/template.js]: I think this is a matter of where the Lambda or CloudFormation code packages `cfn-response`, not anything to do with the aws sdk.  The support ticket I linked also mentions this.

I did a lot of testing but I'm ready to let this go for now.  I think even using a different package version would be more onerous than this pretty simple workaround.  I will open a support case to track it.

**Pull request merged**